src/pydvl/influence/frameworks/torch_differentiable.py

```diff
@@ -33,93 +26,164 @@ def flatten_gradient(grad):
     return torch.cat([el.reshape(-1) for el in grad])
 
 
-class TorchTwiceDifferentiable(TwiceDifferentiable):
+def solve_linear(matrix: torch.Tensor, b: torch.Tensor):
+    """Computes the solution of a square system of linear equations"""
+    return torch.linalg.solve(matrix, b)
+
+
+def as_tensor(a: Any, ensure=True, **kwargs):
```

Comment:
`ensure` seems to be a misnomer, since the conversion is always ensured, but this flag decides whether a warning is issued. Maybe `warn`? (And docstring missing)